packages/perseus-core/src/utils/test-utils.ts

```diff
@@ -1,11 +1,19 @@
-import type {PerseusItem} from "../data-schema";
+import type {PerseusItem, PerseusRenderer} from "../data-schema";
+
+const genericPerseusRenderer: PerseusRenderer = {
```

Comment:
I might call this `blankPerseusRenderer` since it doesn't have any information. `generic` makes me think it takes type arguments or something.

```suggestion
export const blankPerseusRenderer: PerseusRenderer = {
```

I also think it might be slightly clearer to export this value directly rather than exporting a `generateTestPerseusRenderer` function. Consider the difference between

```ts
const testRenderer = {
    ...blankPerseusRenderer,
    widgets: {"group 1": groupWidget},
}
```

and

```ts
const testRenderer = generateTestPerseusRenderer({
    widgets: {"group 1": groupWidget},
})
```

The function call looks like it might be hiding something sophisticated. The object merge obviously has nothing up its sleeve.

Reply:
Renamed from `generic` to `blank`.

I prefer factories over object literals because it allows us to control how objects are created in one central place. For instance if we wanted to change this from a shallow to a deep merge, we'd just update the factory.

My real intent though is to keep one test from mutating the test data another test depends on. Unfortunately I didn't really do it right here, did I? Because you can still mutate nested objects...

I'll fix that.